sdk/test/opentelemetry/sdk/trace/export/in_memory_span_exporter_test.rb

```diff
@@ -7,4 +7,67 @@
 require 'test_helper'
 
 describe OpenTelemetry::SDK::Trace::Export::InMemorySpanExporter do
+  export = OpenTelemetry::SDK::Trace::Export
+
+  let(:span1)   { OpenTelemetry::Trace::Span.new }
+  let(:span2)   { OpenTelemetry::Trace::Span.new }
+
+  let(:subject) { export::InMemorySpanExporter.new }
+
+  it 'accepts an Array of Spans as argument to #export' do
+    subject.export([span1, span2])
+
+    finished_spans = subject.finished_spans
+    finished_spans[0].must_equal span1
+    finished_spans[1].must_equal span2
+  end
+
+  it 'accepts an Enumerable of Spans as argument to #export' do
+    # An anonymous Struct serves as a handy implementor of Enumerable
+    enumerable = Struct.new(:span1, :span2).new
+    enumerable.span1 = span1
+    enumerable.span2 = span2
+
+    subject.export(enumerable)
```

Comment:
minor style comment: In rspec it's usually good practice to use named subjects. in this case I'd just rename subject to exporter so it's easier to read it